client/clientservice/src/request_service.cpp

```diff
@@ -68,10 +71,12 @@ void RequestServiceCallData::sendToConcordClient() {
   bool is_any_request_type = false;
   bft::client::Msg msg;
   if (request_.has_typed_request()) {
-    google::protobuf::Any app_request = request_.typed_request();
-    size_t request_size = app_request.ByteSizeLong();
+    ConcordClientRequest concordRequest;
```

Comment:
To be consistent with the style guide, please rename this to:
```suggestion
    ConcordClientRequest concord_request;
```

Reply:
Updated code